lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -37,17 +38,19 @@ sub new {
 sub init {
     my ($self, $args) = @_;
     $self->SUPER::init($args);
-    $self->{ExpertPartitionerPage}        = Installation::Partitioner::LibstorageNG::v4_3::ExpertPartitionerPage->new({app => YuiRestClient::get_app()});
-    $self->{ClonePartitionsDialog}        = Installation::Partitioner::LibstorageNG::v4_3::ClonePartitionsDialog->new({app => YuiRestClient::get_app()});
-    $self->{CreatePartitionTablePage}     = Installation::Partitioner::LibstorageNG::v4_3::CreatePartitionTablePage->new({app => YuiRestClient::get_app()});
-    $self->{DeletingCurrentDevicesDialog} = Installation::Partitioner::LibstorageNG::v4_3::DeletingCurrentDevicesDialog->new({app => YuiRestClient::get_app()});
```

Comment:
I'm not sure if we should unity all warnings in single dialog, or is it exactly same one?

Reply:
while we don't check the content of the dialog I guess for now with one class should be enough. Or I can leave this as base class and then implement those two to not repeat the code, wdyt?

Reply:
Yeah, base class like pop-up would work, but we can also leave it for later, as we also see a need in some base class for wizard page not to repeat `press_next` method in each page class.